less/index.less

```diff
@@ -53,7 +53,100 @@ table {
     box-sizing: border-box;
 }
 
+//Variables
+@text-color:white;
+
+//mixin
+
+.center-stuff() {
+	display:flex;
+	justify-content:center;
+	align-content: space-between;
+}
+
+.mobile-setting() {
+	flex-direction: column;
+}
+
+
 html, body {
     height: 100%;
+		color:@text-color;
+		background-image: url("https://cdn.pixabay.com/photo/2018/04/25/08/59/super-heroes-3349031_1280.jpg");
```

Comment:
perfect use here, now if you ever want to change the overall color theme of the site, you can change it in 1 place only and not worry.